examples/flutter_gallery/lib/demo/buttons_demo.dart

```diff
@@ -2,6 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
```

Comment:
we should probably not include Cupertino widgets long-term as a separate demo. Instead #8410 should take care of that (since you can switch the OS from the drawer in the gallery, or from hot reload).